.gitignore

```diff
@@ -9,3 +9,4 @@ Portfile~
 *.patch~
 
 .DS_Store
+*.swp
```

Comment:
Please avoid mixing unrelated changes in one commit.

Reply:
It was either that or let another unrelated file into the commit

Reply:
You can add files to Git selectively with `git add` and `git restore --staged` (restore files in the index), then run `git commit` without `-a`.

Vim swap files are temporary, so you could also quit Vim and commit as usual.

I'm not against adding swap file patterns to .gitignore, but it's better done in a separate PR or commit.

Reply:
@cooljeanius can you please take care of this review comment so that we can merge the PR?